src/style.css

```diff
@@ -1,3 +1,66 @@
 body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
   margin: 0;
 }
+
+header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+}
+
+.logo {
+  margin: 10px 0;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  text-transform: uppercase;
+  list-style-type: none;
+}
+
+.nav__item {
+  list-style: none;
+}
```

Comment:
`list-style: none` should be on <ul> tag, not <li>, then `list-style-type: none;` is unnecessary